services/frontend/README.md

```diff
@@ -25,3 +25,9 @@ The ocs endpoint, by default `/ocs`, implements the ownCloud 10 Open Collaborati
 ## Scalability
 
 While the frontend service does not persist any data it does cache `Stat()` responses and user information. Therefore, multiple instances of this service can be spawned in a bigger deployment like when using container orchestration with Kubernetes, when configuring `FRONTEND_OCS_RESOURCE_INFO_CACHE_TYPE=redis` and the related config options.
+
+## Libregraph Service Interactions
```

Comment:
I am not happy with the section title because it is hard to understand what is meant.
What about:

`Define Read-Only Attributes`

Reply:
done 😄